homeassistant/components/hydrawise/coordinator.py

```diff
@@ -40,8 +43,31 @@ async def _async_update_data(self) -> HydrawiseData:
         user = await self.api.get_user()
         controllers = {}
         zones = {}
+        sensors = {}
+        daily_water_use: dict[int, ControllerWaterUseSummary] = {}
         for controller in user.controllers:
             controllers[controller.id] = controller
             for zone in controller.zones:
                 zones[zone.id] = zone
-        return HydrawiseData(user=user, controllers=controllers, zones=zones)
+            for sensor in controller.sensors:
+                sensors[sensor.id] = sensor
+
+            if any(
+                "flow meter" in sensor.model.name.lower()
+                for sensor in controller.sensors
+            ):
+                daily_water_use[controller.id] = await self.api.get_water_use_summary(
+                    controller,
+                    datetime.now(DEFAULT_TIME_ZONE).replace(
```

Comment:
There's a `now()` function in `homeassistant.util.dt` that does the same thing. Maybe just use that instead?

Reply:
Ah. Fixed.